shell/platform/embedder/embedder.h

```diff
@@ -826,6 +826,8 @@ typedef struct {
   double physical_view_inset_bottom;
   /// Left inset of window.
   double physical_view_inset_left;
+  /// The identifier of the display the view is rendering on.
+  size_t display_id;
```

Comment:
Just to double-check, is `size_t` the best type for this? Platform views use `int64_t` for their identifiers.

Reply:
Switching for uint64_t

Reply:
@dnfield I think this was supposed to be `int64_t` instead of `uint64_t`. Let me know if I missed something though!

Reply:
Ah just saw this. I misread int64_t for uint. I'm not sure why this would be signed, but I can make it signed to avoid confusion with platform views in this API.

Reply:
Gotcha, that makes sense. Thanks for checking!